Added some resources for learning Python.

ohkhan opened the pull request.

ohkhan committed:
Added some resources for learning Python.

nelson-liu commented on README.md:
```diff
@@ -27,10 +27,16 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 
 * We recommend you install the [free Anaconda Python distribution](https://www.continuum.io/downloads), which will automatically install Python along with all of the machine learning and scientific computing libraries you need.
 
+* It's a good idea to learn Python's syntax, here are some resources we've used and recommend for learning 'Python for Data Science':
```
I think it's a bit obvious that "it's a good idea to learn Python's syntax" ;) perhaps rephrase to: "Here are some resources for getting started with both the Python language in general and Python for data science specifically"

nelson-liu commented on README.md:
```diff
@@ -27,10 +27,16 @@ Probably the most well known textbook on machine learning. Recommended for mathe
+* It's a good idea to learn Python's syntax, here are some resources we've used and recommend for learning 'Python for Data Science':
+  
+  - [CodeAcademy](https://www.codecademy.com/learn/python)
+  - [PyData Tutorial](https://www.youtube.com/watch?v=SWjnth2_JRY) and associated [notebook](https://github.com/gumption/Python_for_Data_Science/blob/master/3_Python_Basic_Concepts.ipynb)
```
maybe change link text to "Python for Data Science Tutorial @ PyData Seattle 2015"

nelson-liu commented on README.md:
```diff
@@ -27,10 +27,16 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 
   - scikit-learn – Python's primary machine learning library
-  - NumPy – an optimized Python numerical library
+  - NumPy – an optimized Python numerical library, NumPy quickstart can be found [here](https://docs.scipy.org/doc/numpy-dev/user/quickstart.html)
```
change comma to period, and then sentence to  "A quick intro to NumPy can be found [here]"

nelson-liu commented on README.md:
```diff
@@ -17,7 +17,7 @@ Probably the most well known textbook on machine learning. Recommended for mathe
+* UW faculty member [Jake VanderPlas](http://staff.washington.edu/jakevdp/) is the author of O'Reilly's "Python Data Science Handbook". All notebooks and resources used in the book are [posted](https://github.com/jakevdp/PythonDataScienceHandbook/tree/master/notebooks).
 
 ## Software
 
```
I can't comment on L24 for some reason, but it might be good to revise...I feel like Python is the predominant language for machine learning / data science, with R as a language for statistical modeling. Besides, we don't teach anything in R.

ohkhan committed:
Made changes.

ohkhan committed:
fuck R

nelson-liu commented on README.md:
```diff
@@ -17,20 +17,26 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 
 ## Software
 
-**Either Python or R are both excellent languages to know for those getting started in machine learning. Lectures will initially be taught in R.**
```
I still think its worth mentioning that R is an ok language for machine learning, but Python seems to be more on the rise?

nelson-liu commented on README.md:
```diff
@@ -17,20 +17,26 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 
+* Here are some resources for getting started with both the Python language in general and Python for data science specifically:
+  
+  - [CodeAcademy](https://www.codecademy.com/learn/python)
```
CodeAcademy -> codecademy

nelson-liu commented on README.md:
```diff
@@ -17,20 +17,26 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 * Marina also has a [list of books on probability](http://www.stat.washington.edu/courses/stat391/spring15/books.html), for STAT 391 students. 
 
-* UW faculty member [Jake VanderPlas](http://staff.washington.edu/jakevdp/) is working on an O'Reilly book "Python Data Science Handbook". Some previews are [posted](https://www.oreilly.com/people/89c9c-jake-vanderplas).
+* UW faculty member [Jake VanderPlas](http://staff.washington.edu/jakevdp/) is the author of O'Reilly's "Python Data Science Handbook". All notebooks and resources used in the book are [posted](https://github.com/jakevdp/PythonDataScienceHandbook/tree/master/notebooks).
```
maybe add a link to the book? http://shop.oreilly.com/product/0636920034919.do

nelson-liu reviewed (changes requested):
thanks for addressing the changes; just some last leftover things.

ohkhan committed:
Appeal to Python vs R debate

Turned it into a QA thing, the answer I gave is very general and there's definitely way more to it.

nelson-liu commented on README.md:
```diff
@@ -17,20 +17,26 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 ## Software
 
-**Either Python or R are both excellent languages to know for those getting started in machine learning. Lectures will initially be taught in R.**
+**Q: Should I use Python or R? A: It depends on what you're doing, both langauges have their strengths; R is arguably better for Statistical Modeling, but for general machine learning/data science Python has a variety of robust libraries all at different levels of abstraction. Lectures will be taught in Python.**
```
sorry last nitpick, can you lowercase statistical modeling. else LGTM

nelson-liu replied:
nevermind ill just fix in master. thanks for the patch.
